roles/sysstat/tasks/common.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Install packages
+  apt:
+    pkg: "{{ item }}"
+    state: latest
+    force: yes
+  with_items:
```

Comment:
Not sure either. Maybe for certbot ?

Some are needed to build python I think. So we should move them in the python role.